app/views/advertiser_dashboards/show.html.erb

```diff
@@ -14,6 +14,10 @@
 
 <main id="content" role="main" class="bg-light">
   <div class="container space-1">
+    
+    <% if authorized_user.can_admin_system? || current_user != true_user %>
+      <small class="text-center d-block pb-3"><%= link_to "open metabase dashboard in new window", "https://metabase.codefund.io/dashboard/10?date_filter=thismonth&advertiser_user_id=#{current_user.id}", class: "text-link", target: "_blank" %></small>
+    <% end %>
```

Comment:
Is `current_user != true_user` too permissive here?

Reply:
Only admins can masquerade right? Is there a better way?

Reply:
Yes. This might be a better solution. 

```
AuthorizedUser.new(true_user).can_admin_system?
```